dist/pf-relative.js

```diff
@@ -12,6 +12,14 @@ angular.module('pf-relative', [])
     var relative = new RelativeDate(scope.date);
     element.attr('datetime', scope.date);
     relative.onTick(element.text.bind(element));
+
+    scope.$watch('date', function (newDate) {
```

Comment:
scope.date holds a string to the value itself, we can't `$watch` it.

:+1: for removing the anonymous function